### ISSUE TITLE
Ask For Credentials - Script Fails When Large Number Of Devices

### ISSUE THREAD

**Issue by Alexpf20210007** [CiscoTestAutomation/unicon.plugins#60]
I am using the ask feature to get the password info at the beginning of the script so that I do not have to worry about storing it in a text, csv, or yaml file. I ran the exact same script with the username and password in the YAML file for 100 devices and it is successful. When I remove all of the username and passwords and put the below info in script, it fails at different devices. When I reduced the number of devices in the script, it was successful.
testbed:
    credentials:
        default:
            password: "%ASK{Password:}"
            username: TESTUSER
        enable:
            password: "%ASK{Enable_Password:}"

**Comment by stevedhn** [comment]
Hi, thanks for your finding, we'd like to welcome the user to contribute to the fix, so feel free to contribute and if you need some helps, please let us know.

**Comment by stevedhn** [comment]
Hi, what files you're refering here? you still can access to unicon plugins repos right? https://github.com/CiscoTestAutomation/unicon.plugins

**Comment by Alexpf20210007** [comment]
Yes, I was thinking that some of the base files that I call are not part of the open source repo. I can look into it and get back with you.

**Comment by stevedhn** [comment]
ok great Alex, you should be able to access to all the files on unicon plugins, 
You probably know this, but just make sure when you contribute, please make a branch out of the original master branch of unicon plugins, not from your forked repos.
I'll close the issue for now, once you are free to contribute, please create a PR and assign me so that we can follow up.
